### PR TITLE
LUI-126: Wildcard selector * is being used in openmrs.css 

### PR DESCRIPTION
**Pull request by Snaddyvitch-Dispenser**
https://issues.openmrs.org/browse/LUI-126

Fixes Use Of Wildcard Selector and Moves It All to the main HTML {} Tag

### CODE REVIEW

File: omod/src/main/webapp/resources/css/openmrs.css [hunk 1]
```diff
@@ -1,4 +1,9 @@
+/*Assigned Font to Html Instead Of * To Fix Issue Where Other Fonts May Get Overriden
```
Comment (gayanW) [inline omod/src/main/webapp/resources/css/openmrs.css]: Can you remove the comments. We don't need those. They should ideally go in your commit or PR description.

File: omod/src/main/webapp/resources/css/openmrs.css [hunk 2]
```diff
@@ -78,15 +83,12 @@ img.bar-round-reduced50 {
 		color: #ffffff;
 	}
 
+
+/*Note: Is Using Wildcard Selector, But Uses Other Selectors Alongside.*/
```
Comment (gayanW) [inline omod/src/main/webapp/resources/css/openmrs.css]: Remove this as well.

**Comment by gayanW** [comment]
Also carefully read these sections in [Pull Request Tips](https://wiki.openmrs.org/display/docs/Pull+Request+Tips):

- Branch name == ticket name.
- Squash! (you can do this once everything is fixed)

**Comment by Snaddyvitch-Dispenser** [comment]
Moved, Reason: Changing Target Branch

**Pull request closed without merging**